hw/ip/hmac/data/hmac_testplan.hjson

```diff
@@ -97,29 +97,33 @@
     {
       name: save_and_restore
       desc: '''Verify save & restore, which allows SW to switch between different parallel message streams.
-
-               TODO(lowRISC/opentitan#21307)
+            - This feature is randomly exercised during most of the tests.
+            - When triggered it could go into 3 different scenarios:
+              - Stop and Continue without Saving and Restoring
+              - Save and Restore with same context
+              - Save and Restore with different contexts
             '''
       stage: V2
-      tests: []
+      tests: ["hmac_smoke", "hmac_long_msg", "hmac_back_pressure", "hmac_burst_wr",
+              "hmac_stress_all", "hmac_datapath_stress", "hmac_error"]
     }
     {
       name: fifo_empty_status_interrupt
       desc: '''Verify the FIFO empty status interrupt.
 
-               TODO(lowRISC/opentitan#21815)
+               TODO(lowRISC/opentitan#21815) -> moved into M5
             '''
       stage: V2
       tests: []
     }
     {
       name: wide_digest_configurable_key_length
       desc: '''Verify wider (SHA-384 and SHA-512) digests as well as configurable key lengths.
-
-               TODO(lowRISC/opentitan#22102)
+            - This feature is randomly exercised during most of the tests.
             '''
       stage: V2
-      tests: []
+      tests: ["hmac_smoke", "hmac_long_msg", "hmac_back_pressure", "hmac_burst_wr",
+              "hmac_stress_all", "hmac_datapath_stress", "hmac_error", "hmac_wipe_secret"]
```

Comment:
+ "hmac_test_sha_vectors" once  https://github.com/lowRISC/opentitan/pull/23541 is merged.

Reply:
I'll add this test as well, but not `hmac_test_hmac_vectors`? It shouldn't be part of the same? I know it's not in the PR #23541 but it'll probably be in a future one.

Reply:
Yes, it will be in a future one, planned for this weekend/early next week if we are to to support test vectors with all the different key lengths for the different digest sizes.